object_database/web/content/components/Grid.js

```diff
@@ -70,12 +98,34 @@ class Grid extends Component {
                     ])
                 );
             });
-        } else {
-            return []
         }
     }
 
-    _makeHeaderElements(){
+    _makeReplacementRowElements(){
+        return this.getReplacementElementsFor('child').map((row, rowIdx) => {
```

Comment:
not sure if this pertains the same in this model, but you can have grids with no children, and that's not supposed to throw an exception.

Reply:
I see. This should be a one line change. Let me try it out here and if it works I'll push.